### DISCUSSION TITLE
类名里面好像有些类似哈希值的东西，这些值会随着更新而改变吗

### DISCUSSION THREAD

**Discussion by ziJinChampion** [ant-design/ant-design#43818] [Q&A] [answered]
:where(.css-dev-only-do-not-override-10ud2r5).ant-collapse 
就比如这个类名，“10ud2r5”这个值非常像是一个会随着更新而改变的值，而我又需要重制这个类名的样式，以后随着更新这个值改变之后样式重制是不是就失效了，我当前的版本是5.6.4

**Comment by zombieJ** [comment, marked as answer]
`css-dev-only-do-not-override` 的意思就是 `不要用我`。生产环境下，这个值和开发环境也会不一样。你要使用的应该是后面的那个 `.ant-collapse`，这个才是提供给开发者覆盖用的。

**Reply by ziJinChampion** [reply]
好的谢谢
